internal/ethapi/api.go

```diff
@@ -295,6 +295,7 @@ func (s *PrivateAccountAPI) DeriveAccount(url string, path string, pin *bool) (a
 func (s *PrivateAccountAPI) NewAccount(password string) (common.Address, error) {
 	acc, err := fetchKeystore(s.am).NewAccount(password)
 	if err == nil {
+		log.Info("Your new key is generated. Please backup the key file", "path", acc.URL.Path)
```

Comment:
Nit, logs should be shorter than 40 character to keep them aligned. Perhaps let's split there into 3? cc @holiman ?

```
log.Info("Your new key is generated", "address", /* dunno what goes here */)
log.Warn("Please backup your key file!", "path", acc.URL.Path)
log.Warn("Please backup your password!")
```

Warn makes sure it's a bit more visible. Open for opposition.

Reply:
If we do go down this path, do it for the other occurrences too.

Reply:
Sure, sounds good